proxies/nodejs/app.ts

```diff
@@ -1,20 +1,53 @@
-import { initialize } from '@devcycle/nodejs-server-sdk'
+import { DVCClient } from '@devcycle/nodejs-server-sdk'
 import Koa from 'koa'
+import Router from 'koa-router'
+import KoaBody  from 'koa-body'
 
-async function start() {
-    await initialize('<TOKEN>').onClientInitialized()
+const data = {
+  clients: {},
+  users: {},
+  commandResults: {}
+}
+
+const handleClient = async (ctx) => {
+  const body = ctx.request.body
+  if (body.clientId === undefined) {
+    ctx.status = 400 
+    ctx.body = "Invalid request: Missing clientId"
+  } else if (!body.sdkKey) {
+    ctx.status = 400 
+    ctx.body = "Invalid request: Missing sdkKey"
```

Comment:
do you want the api to validate this or do you want to just pass it verbatim to the SDK and see what happens? otherwise how do we test how the SDK handles not providing an SDK token?